src/bin/rdcore/cmdline.rs

```diff
@@ -115,4 +115,7 @@ pub struct VerifyUniqueFsLabelConfig {
     /// Filesystem's label
     #[structopt(value_name = "LABEL")]
     pub label: String,
+
+    #[structopt(long)]
+    pub reread_pt: bool,
```

Comment:
Needs a docstring for display in `--help`.  Also, FWIW, `blockdev(8)` calls this `--rereadpt`.